Library/Homebrew/dev-cmd/livecheck.rb

```diff
@@ -95,7 +95,7 @@ def livecheck
                         .reject { |line| line.start_with?("#") || line.blank? }
                         .map(&:strip)
 
-        named_args = T.unsafe(CLI::NamedArgs).new(*names, parent: args)
+        named_args = CLI::NamedArgs.new(*names, parent: args)
```

Comment:
So are splats now supported here, or what changed?

Reply:
Not supported, we just decided to give up 🙈 
https://github.com/Homebrew/brew/blob/master/Library/Homebrew/sorbet/config#L5

Reply:
I see. 😄